usaspending_api/accounts/serializers.py

```diff
@@ -126,7 +126,6 @@ class ObjectClassFinancialSpendingSerializer(serializers.Serializer):
 
 class AgenciesFinancialBalancesSerializer(serializers.Serializer):
 
-    fiscal_year = serializers.IntegerField()
```

Comment:
Since we're removing it from the serializer, do we want to alter the queryset to no longer include `.values('fiscal_year')`?